### PR TITLE
[Update 1.2.0] - updating release notes, changelog, package.json

### PR DESCRIPTION
**Pull request by bgoddar**
Updating CHANGELOG.md, package.json and the release notes to reflect completed work items for version 1.2.0

### CODE REVIEW

**Comment by bgoddar** [comment]
Changelog:
image

**Comment by bgoddar** [comment]
Welcome:
image

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "vscode-edge-devtools",
     "displayName": "Microsoft Edge Tools for VS Code",
     "description": "Use the Microsoft Edge Tools from within VS Code to see your site's runtime HTML structure, alter its layout, fix styling issues as well as see your site's network requests.",
-    "version": "1.1.9",
+    "version": "1.1.10",
```
Comment (mliao95) [inline package.json:5]: I feel like the version update itself is big enough to update the minor to `1.2.0`.  Any thoughts on this?

**Review by mliao95 — COMMENTED** [review]
Versioning comment.

File: package.json (hunk 1 above)
Comment (bgoddar) [reply package.json:5]: Seems reasonable to me. I'm not really sure what our versioning story is

Comment (mliao95) [reply package.json:5]: We haven't really fleshed it out too much (which we should), but I would want to somewhat follow these standards:

```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
```

Comment (bgoddar) [reply package.json]: Updated (images in conversation show old version number, but they have been updated in code)

**Review by mliao95 — APPROVED** [review]
LGTM

**Review by mliao95 — COMMENTED** [review]
LGTM

**Pull request merged**